sql/mob_droplist.sql

```diff
@@ -10066,8 +10066,7 @@ INSERT INTO `mob_droplist` VALUES (1217,0,0,1000,4374,500); -- Sleepshroom (50.0
 INSERT INTO `mob_droplist` VALUES (1217,0,0,1000,4374,250); -- Sleepshroom (25.0%)
 INSERT INTO `mob_droplist` VALUES (1217,2,0,1000,4374,0);   -- Sleepshroom (Steal)
 
--- ZoneID:  25 - Gration
-INSERT INTO `mob_droplist` VALUES (1218,0,0,1000,12360,500); -- Tatami Shield (50.0%)
+-- 1218 Available
```

Comment:
This may get lost in the spam, usually available slots like this are noted at the end of file

Reply:
Oh? I don't usually make mobs so you can ignore me then